woocommerce/woocommerce-ios
# Shipping Labels: displays a list of products associated with the order under a single package

Part of #2974 

## Description
In this PR, as part of Shipping Labels M2, I implemented the first screen of the Package Details flow, where we display a list of products associated with the order under a single package. The multipackage version will be part of the next milestone.
Since we are starting to introduce more and more components in SwiftUI, this view lent itself particularly well to introducing new components implemented in SwiftUI.
Since the view needs to show a list of different cells that in some cases we already implemented in UIKit, I think it was better to implement these cells from scratch and make them generic components in SwiftUI that can be reused in the future. For this reason, I created a folder called `SwiftUI Components` under the main folder `ReusableViews` that will collect all the reusable SwiftUI components. For the moment, these components are a header and rows that we are going to use in the package detail view.
For every component, a preview is available. In that way, it's easy to understand the appearance of the component.

For the moment, the view has no logic if not for showing the items list.

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail.
3. Press the button "Create Shipping Label".
4. Press the button "Continue" in Create Shipping Label form, under the **Ship From** cell. Confirm the address.
5. Press the button "Continue" in Create Shipping Label form, under the **Ship To** cell. Confirm the address.
6. Press the button "Continue" in Create Shipping Label form, under the **Package Details** cell.
7. You should see the new screen that shows the Package Details screen and all the items that should be shipped (the number of items and the description of the cell for the moment is wrong, and will be handled in another PR).

## Screenshots
| Light mode             |  Dark mode |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 12 Pro - 2021-03-30 at 12 19 21](https://user-images.githubusercontent.com/495617/112975132-969b3800-9153-11eb-9fe0-083c74bd564a.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-30 at 12 19 10](https://user-images.githubusercontent.com/495617/112975140-9864fb80-9153-11eb-9fc1-968f4164cd52.png)


Update release notes:
